arangod/Agency/FailedFollower.cpp

```diff
@@ -166,8 +166,25 @@ bool FailedFollower::start(bool& aborts) {
     return false;
   }
 
-  // Get proper replacement
-  _to = randomIdleAvailableServer(_snapshot, planned);
+  // Exclude servers in failoverCandidates for some clone and those in Plan:
```

Comment:
the comment here says we are excluding servers in failoverCandidates, but line 177 looks to me as we would be excluding servers that are _not_ in failoverCandidates. is this intentional?